.github/workflows/ci-sharktank.yml

```diff
@@ -22,45 +22,6 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test_punet:
-    name: "Integration Tests - punet"
-    runs-on: nodai-amdgpu-mi250-x86-64
```

Comment:
This base workflow should use standard github-hosted runners if possible. The `test_with_data` job is already a stretch.